scripts/outer/kubectl-kubestellar-ensure-location

```diff
@@ -114,10 +101,7 @@ set -e
 
 echo "--- current directory is $PWD"
 
-imw_space_config="${PWD}/temp-space-config/spaceprovider-default-${imw}"
-kubectl-kubestellar-get-config-for-space --space-name ${imw} --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --output $imw_space_config
-
-if ! kubectl "${kubectl_flags[@]}" --kubeconfig $imw_space_config get synctargets.edge.kubestellar.io "$objname" &> /dev/null; then
+if ! kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$objname" &> /dev/null; then
```

Comment:
Do we need to run these kubectl commands with a particular kubeconfig?  If so how do you make sure it happens?  By assuming the --kubeconfig <config> is passed on the kubectl_flags?

Reply:
The caller gets to decide how to pass in the desired kubeconfig; as a kubectl plugin, all the usual ways are available (and as a kubectl plugin all the usual ways SHOULD be supported):
- via `--kubeconfig` on the command line;
- referenced in `$KUBECONFIG`;
- in `~/.kube/config`.